Ghidra/Processors/68000/data/languages/68000.sinc

```diff
@@ -1719,7 +1719,7 @@ ptestLevel: "#"^mregn	is mregn  { export *[const]:1 mregn; }
 
 :rts				is opbig=0x4e & op37=14 & op02=5				{ PC = *SP; SP = SP+4; return [PC]; }
 
-:sbcd Tyb,Txb			is op=8 & op48=16 & Txb & Tyb
+:sbcd Txb,Tyb			is op=8 & op48=16 & Txb & Tyb
```

Comment:
Disassemble an instruction in ghidra and objdump.  Is display the same?  All this does is break the display right?

Reply:
oh wait, this is true

Reply:
Sorry, I'm kinda rusty with SLEIGH. Also, it seems Txb corresponds to Dy in the manual and vice versa (?). Does this new modification make sense?